src/main/java/com/example/backend/repositories/CardRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.example.backend.repositories;
+
+import com.example.backend.entity.Card;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CardRepository extends JpaRepository<Card, String> {
```

Comment:
Card entity har en Long som primary key, lad os bruge den I stedet for String :)